app/src/components/map/Map.vue

```diff
@@ -80,13 +81,48 @@ export default defineComponent({
       })),
     );
 
+    const selectedMarker = toRef(props, 'selectedMarker');
+
+    const vehicle = computed(() => vehicles.value[selectedMarker.value.id]);
+
+    const trip = computed(() => {
+      if (!trips.value || !vehicle.value) {
+        return null;
+      }
+      return trips.value[vehicle.value.tripId];
```

Comment:
@lukashass To get a list of all stops in the longterm / a followup PR we could try to look for all other busses of the same line and add their stops as well.

Reply:
#167